From d1ba94ac41f3864a5068600b8e5ce4a8bce2bc79 Mon Sep 17 00:00:00 2001
From: Giacomo Travaglini <giacomo.travaglini@arm.com>
Date: Fri, 18 Mar 2022 14:45:34 +0000
Subject: [PATCH 435/757] util: Remove python3-six package from dockerfiles

python six package is not used in gem5 as we don't support
python2 anymore

Change-Id: I25a682842ad00c0b5e09c9cb4ea6efac5114da6d
Signed-off-by: Giacomo Travaglini <giacomo.travaglini@arm.com>
Reviewed-on: https://gem5-review.googlesource.com/c/public/gem5/+/57969
Reviewed-by: Jason Lowe-Power <power.jg@gmail.com>
Reviewed-by: Bobby Bruce <bbruce@ucdavis.edu>
Maintainer: Bobby Bruce <bbruce@ucdavis.edu>
Tested-by: kokoro <noreply+kokoro@google.com>
---
 site_scons/site_init.py                                   | 2 +-
 util/dockerfiles/gcn-gpu/Dockerfile                       | 2 +-
 util/dockerfiles/sst-11.1.0/Dockerfile                    | 2 +-
 util/dockerfiles/ubuntu-18.04_all-dependencies/Dockerfile | 2 +-
 util/dockerfiles/ubuntu-18.04_clang-version/Dockerfile    | 2 +-
 util/dockerfiles/ubuntu-18.04_gcc-version/Dockerfile      | 2 +-
 util/dockerfiles/ubuntu-20.04_all-dependencies/Dockerfile | 4 ++--
 util/dockerfiles/ubuntu-20.04_clang-version/Dockerfile    | 2 +-
 util/dockerfiles/ubuntu-20.04_gcc-version-11/Dockerfile   | 2 +-
 util/dockerfiles/ubuntu-20.04_gcc-version/Dockerfile      | 2 +-
 10 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/site_scons/site_init.py b/site_scons/site_init.py
index 6a03d84..5eeb290 100644
--- a/site_scons/site_init.py
+++ b/site_scons/site_init.py
@@ -63,7 +63,7 @@ The following are steps to compile gem5 in Python 3 environment,
 *Step 1*: ensure Python 3 is installed. On Ubuntu like systems, you can try \
 this command:
 
-    sudo apt-get install python3 python3-six python-is-python3 python3-pydot
+    sudo apt-get install python3 python-is-python3 python3-pydot
 
 To run Python 3 from a container, you can try the Docker files in \
 util/dockerfiles folder.
diff --git a/util/dockerfiles/gcn-gpu/Dockerfile b/util/dockerfiles/gcn-gpu/Dockerfile
index 8dd1b1b..dd4e658 100644
--- a/util/dockerfiles/gcn-gpu/Dockerfile
+++ b/util/dockerfiles/gcn-gpu/Dockerfile
@@ -29,7 +29,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install build-essential git m4 scons zlib1g zlib1g-dev \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3-six python-is-python3 doxygen libboost-all-dev \
+    python3-dev python-is-python3 doxygen libboost-all-dev \
     libhdf5-serial-dev python3-pydot libpng-dev libelf-dev pkg-config
 
 # Requirements for ROCm
diff --git a/util/dockerfiles/sst-11.1.0/Dockerfile b/util/dockerfiles/sst-11.1.0/Dockerfile
index f5fc0f2..5103ffe 100644
--- a/util/dockerfiles/sst-11.1.0/Dockerfile
+++ b/util/dockerfiles/sst-11.1.0/Dockerfile
@@ -31,7 +31,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install build-essential git m4 scons zlib1g zlib1g-dev \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3-six python-is-python3 doxygen libboost-all-dev \
+    python3-dev python-is-python3 doxygen libboost-all-dev \
     libhdf5-serial-dev python3-pydot libpng-dev libelf-dev pkg-config pip \
     python3-venv wget
 
diff --git a/util/dockerfiles/ubuntu-18.04_all-dependencies/Dockerfile b/util/dockerfiles/ubuntu-18.04_all-dependencies/Dockerfile
index 5a9fd8d..adcc0b0 100644
--- a/util/dockerfiles/ubuntu-18.04_all-dependencies/Dockerfile
+++ b/util/dockerfiles/ubuntu-18.04_all-dependencies/Dockerfile
@@ -30,7 +30,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install build-essential git m4 scons zlib1g zlib1g-dev \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3 python3-six doxygen libboost-all-dev \
+    python3-dev python3 doxygen libboost-all-dev \
     libhdf5-serial-dev python3-pydot libpng-dev libelf-dev pkg-config \
     python3-pip python3-venv
 
diff --git a/util/dockerfiles/ubuntu-18.04_clang-version/Dockerfile b/util/dockerfiles/ubuntu-18.04_clang-version/Dockerfile
index 869a2c1..4575ef9 100644
--- a/util/dockerfiles/ubuntu-18.04_clang-version/Dockerfile
+++ b/util/dockerfiles/ubuntu-18.04_clang-version/Dockerfile
@@ -40,7 +40,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install git m4 scons zlib1g zlib1g-dev clang-${version} \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3 python3-six doxygen make
+    python3-dev python3 doxygen make
 
 RUN apt-get --purge -y remove gcc
 
diff --git a/util/dockerfiles/ubuntu-18.04_gcc-version/Dockerfile b/util/dockerfiles/ubuntu-18.04_gcc-version/Dockerfile
index 1723fd9..cdb80b7 100644
--- a/util/dockerfiles/ubuntu-18.04_gcc-version/Dockerfile
+++ b/util/dockerfiles/ubuntu-18.04_gcc-version/Dockerfile
@@ -37,7 +37,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install git m4 scons zlib1g zlib1g-dev gcc-multilib \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3 python3-six doxygen wget zip gcc-${version} \
+    python3-dev python3 doxygen wget zip gcc-${version} \
     g++-${version} make
 
 RUN update-alternatives --install \
diff --git a/util/dockerfiles/ubuntu-20.04_all-dependencies/Dockerfile b/util/dockerfiles/ubuntu-20.04_all-dependencies/Dockerfile
index 76d3011..3fef444 100644
--- a/util/dockerfiles/ubuntu-20.04_all-dependencies/Dockerfile
+++ b/util/dockerfiles/ubuntu-20.04_all-dependencies/Dockerfile
@@ -31,8 +31,8 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install build-essential git m4 scons zlib1g zlib1g-dev \
     libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev \
-    python3-dev python3-six python-is-python3 doxygen libboost-all-dev \
+    python3-dev python-is-python3 doxygen libboost-all-dev \
     libhdf5-serial-dev python3-pydot libpng-dev libelf-dev pkg-config pip \
     python3-venv
 
-RUN pip install mypy
\ No newline at end of file
+RUN pip install mypy
diff --git a/util/dockerfiles/ubuntu-20.04_clang-version/Dockerfile b/util/dockerfiles/ubuntu-20.04_clang-version/Dockerfile
index 534291b..ceb11ab 100644
--- a/util/dockerfiles/ubuntu-20.04_clang-version/Dockerfile
+++ b/util/dockerfiles/ubuntu-20.04_clang-version/Dockerfile
@@ -40,7 +40,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install git m4 scons zlib1g zlib1g-dev libprotobuf-dev \
     protobuf-compiler libprotoc-dev libgoogle-perftools-dev python3-dev \
-    python3-six python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
+    python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
     python3-pydot libpng-dev clang-${version} make
 
 RUN apt-get --purge -y remove gcc
diff --git a/util/dockerfiles/ubuntu-20.04_gcc-version-11/Dockerfile b/util/dockerfiles/ubuntu-20.04_gcc-version-11/Dockerfile
index 4effd9c..8d570f0 100644
--- a/util/dockerfiles/ubuntu-20.04_gcc-version-11/Dockerfile
+++ b/util/dockerfiles/ubuntu-20.04_gcc-version-11/Dockerfile
@@ -35,7 +35,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install git m4 scons zlib1g zlib1g-dev libprotobuf-dev \
     protobuf-compiler libprotoc-dev libgoogle-perftools-dev python3-dev \
-    python3-six python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
+    python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
     python3-pydot libpng-dev make software-properties-common
 
 RUN add-apt-repository \
diff --git a/util/dockerfiles/ubuntu-20.04_gcc-version/Dockerfile b/util/dockerfiles/ubuntu-20.04_gcc-version/Dockerfile
index 923fe63..c121fe0 100644
--- a/util/dockerfiles/ubuntu-20.04_gcc-version/Dockerfile
+++ b/util/dockerfiles/ubuntu-20.04_gcc-version/Dockerfile
@@ -37,7 +37,7 @@ RUN apt -y update
 RUN apt -y upgrade
 RUN apt -y install git m4 scons zlib1g zlib1g-dev libprotobuf-dev \
     protobuf-compiler libprotoc-dev libgoogle-perftools-dev python3-dev \
-    python3-six python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
+    python-is-python3 doxygen libboost-all-dev libhdf5-serial-dev \
     python3-pydot libpng-dev gcc-${version} g++-${version} make
 
 RUN update-alternatives --install \
-- 
1.8.3.1

